public/locales/de-DE/grafana.json

```diff
@@ -215,6 +215,12 @@
       "unmark-favorite": "Markierung als Favorit entfernen"
     }
   },
+  "dashboard-settings": {
+    "json-editor-settings": {
+      "save-button": "Änderungen speichern",
```

Comment:
please don't add translations to this file. these should be autogenerated when running `yarn i18n:extract` and should all be blank _except_ for the english and pseudo translations file.

we're not currently accepting contributions for the actual translation strings themselves. you can see this in the instructions [here](https://github.com/grafana/grafana/blob/main/contribute/internationalization.md#how-to-add-a-new-translation-phrase:~:text=Please%20note%3A%20We%20do%20not%20currently%20accept%20contributions%20for%20translations.%20Please%20do%20not%20submit%20pull%20requests%20translating%20grafana.json%20files%20%2D%20they%20will%20be%20rejected.%20We%20do%20accept%20contributions%20to%20mark%20up%20phrases%20for%20translation.)